noxfile.py

```diff
@@ -773,13 +775,21 @@ def notebook(session: nox.Session):
             *notebooks,
         )
 
-        # Run self-contained notebooks in single session.run
-        # achieve parallelization via -n
-        session.run(
-            *pytest_command,
-            "-nauto",
-            *notebooks,
-        )
+        # Run notebooks in parallel session.run's, since each notebook
+        # takes an environment variable for performance logging
+        processes = []
+        for notebook in notebooks:
+            session.env[logging_name_env_var] = os.path.basename(notebook)
+            process = Process(
+                target=session.run,
+                args=(*pytest_command, notebook),
+            )
+            process.start()
```

Comment:
Concurrent execution of these tests could introduce performance inconsistencies because of resource contention. For reliable results, it's best to run them in isolation. Additionally, could you examine the logs from parallel runs? My concern is that they may be interleaved, making analysis difficult.

Reply:
Discussed this offline. Based on this being nbmake not regular pytest, and based on the way we are running nbmake sessions in parallel below, this should be okay considering the parameterization of the logging requirement. We also checked the logs from parallel runs.